### ISSUE TITLE
Multiple instances

### ISSUE THREAD

**Issue by sicherist** [cboxdoerfer/fsearch#270]
Hi,

is it possible to start multiple instances of fsearch?

The reason I'm asking is that I have fsearch on a hotkey in KDE. But if fsearch is already running in the background the hotkey will neither bring the background instance to the foreground, nor will it start a new instance, so I will have to manually switch to the window.

**Comment by cboxdoerfer** [comment]
I have to test this on KDE, on GNOME it works as expected: Using the shortcut either activates the current window if FSearch is already running or launches it otherwise.

**Comment by sicherist** [comment]
Thanks. On Kubuntu 21.04 it is like this: When the window is minimized the shortcut opens it. But when the window is in the background the shortcut doesn't bring it to the front.

**Comment by cboxdoerfer** [comment]
Yes, I can confirm this behavior on KDE. It seems KDE's focus stealing mechanism gets triggered here, so setting `Settings -> Window Management -> Window Behavior -> Focus stealing prevention` to `None` fixes the issue on my system. However I'm not yet sure if this is intentional, or if there's something else I can do in FSearch to force the window on top. I'll need to investigate this further.

**Comment by sicherist** [comment]
Thank you! The workaround with diabling focus stealing protection ("Verhindern unerwünschter Aktivierung") works for me too.

**Comment by zDEFz** [comment]
Would it be possible to launch multiple instances with different search scopes?
